pkg/engine/engine.go

```diff
@@ -308,145 +345,200 @@ func (engine *GameEngine) SendGetLegalMovesBatch(concreteRequests []*GetLegalMov
 //
 // If a request fails before reaching a worker, a `SyncResponse` type will be
```

Comment:
the 'be' is repeated in the next line
```suggestion
// If a request fails before reaching a worker, a `SyncResponse` type will
```